man/set-content-if-empty.md

```diff
@@ -0,0 +1,27 @@
+## set content if empty
+
+Creates a function that when passed to d3's `selection.select` operator will conditionally set the HTML content of a node (only if it is empty) and select the first child DOM element it finds.
```

Comment:
Yes, the current implementation allows you to use templates with multiple top level nodes.

```javascript
 const template = `
<header class="headings">
  <h2 class="greeting">hello</h2>
</header>
<div class="frame">
  <div class="content" />
</div>
`

target
  .select(setContentIfEmpty(template))
    .style('background-color', 'plum')

target
  .select('.content')
     .text(textContent)

 ```
 
 I think we should allow the user to do this, even if it is not the common case.